src/shared/components/sign/sign-view.html

```diff
@@ -50,7 +50,7 @@
     	
         <p ng-if="!ctrl.loaded" flex>Loading schools...</p>
 
-		<md-input-container flex layout="column" ng-if="ctrl.loaded &amp;&amp; currentUser.yearOfBirth >= 1996 &amp;&amp; currentUser.yearOfBirth <= 2004">
+		<md-input-container flex layout="column" ng-if="currentYear - currentUser.yearOfBirth <= 20 &amp;&amp; currentYear - currentUser.yearOfBirth  >= 10">
```

Comment:
I think this should be okay. As what I observe from my school, in Junior College Year 2, the year of birth ranges from 1995 to 1998. Adding 6 to 1998 will be 2004.